tests/testthat/test-validator.R

```diff
@@ -59,7 +59,7 @@ test_that("simple case works", {
   expect_true(v("{hello: 'world'}"))
 
   v <- json_validator("schema2.json", "ajv")
-  expect_error(v("{}", error = TRUE), "hello")
+  expect_condition(v("{}", error = TRUE), "hello", class = "validation_error")
```

Comment:
could you just keep that as `expect_error` - it's not clear to me what the difference in these expectation functions is in particular and we do want an error thrown here